workflow/schemas/repository.json

```diff
@@ -1,6 +1,7 @@
 {
   "title": "Dataset Submission Form",
```

Comment:
should this be repository submission form?

Reply:
^